quic/s2n-quic-platform/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "s2n-quic-platform"
-version = "0.18.1"
+version = "0.18.2"
```

Comment:
I guess this ones should technically be a minor bump since the feature was in s2n-quic-platform?

Reply:
Yeah... I wasn't sure if we considered it in s2n-quic or s2n-quic-platform. But I can do a minor bump for platform.